pkg/resources/label.go

```diff
@@ -12,3 +14,20 @@ func labelsForHawtio(name string) map[string]string {
 		labelResourceKey: name,
 	}
 }
+
+// example: label1=value1;label2=value2
+func parseLabels(text string) map[string]string {
+	labels := make(map[string]string)
```

Comment:
It may be better to call methods from the k8s.io/apimachinery/pkg/labels package, like ConvertSelectorToLabelsMap that does the parsing and the validation, and comply with the usual string serialisation form for label selectors.

Reply:
Thanks! How about the new changes?